src/components/elements/EventsForMonth.jsx

```diff
@@ -15,36 +16,67 @@ const styleClasses = {
   `
 }
 
+function RenderEventType(data) {
+  const eventStyles = `inline-block mr-1 pl-2 pr-2 rounded bg-event-tags-${data.data} text-white`
+
+  return <span className={eventStyles}>{data.data}</span>
+}
+
 function formatEventTags(tags) {
   if (tags === null || tags === undefined) return [];
 
   return tags.trim().split(',').filter(tag => tag !== '');
 }
 
+function DateRange(event) {
```

Comment:
This was to format the dates to read 'September 6 - 7, 2022' instead of 'September 6, 2022 - September 6, 2022.' If it's a different month, it will go ahead and render the whole date range though, so it's not 'September 29 - 2, 2022.'